app/code/Magento/Captcha/Observer/CheckUserLoginObserver.php

```diff
@@ -143,10 +155,10 @@ public function execute(\Magento\Framework\Event\Observer $observer)
                 try {
                     $customer = $this->getCustomerRepository()->get($login);
                     $this->getAuthentication()->processAuthenticationFailure($customer->getId());
-                } catch (NoSuchEntityException $e) {
```

Comment:
Hi,
This change is not related to this PR, could you revert this part as it was earlier?
If static tests complains about it - please add ignore for this case

Reply:
Hi, @ihor-sviziev! Thank you for notice!

Do you know the ignore command for this particular case? Static test error was something like "Empty catch statement..."

Thank you!

Reply:
Try to use following:
https://github.com/magento/magento2/blob/ee6bc3729e85ed1f1b5cd7c0eb9a3800bdd274c5/dev/tests/integration/testsuite/Magento/SalesRule/_files/cart_rule_100_percent_off_rollback.php#L25

Reply:
> Try to use following:

tried

Let's see if your magic can help here, @ihor-sviziev

Thank you!